timemachine/fe/absolute_hydration.py

```diff
@@ -131,6 +148,7 @@ def set_up_ahfe_system_for_smc(
     mol, n_walkers, n_windows, n_md_steps, resample_thresh, seed=2022, ff=None, num_workers=None
 ):
     """define initial samples, lambdas schedule, propagate fxn, log_prob fxn, resample fxn"""
+    n_windows = n_windows or 16
```

Comment:
16 is arbitrary here, need to do some tests w/free solv to determine a good default

Reply:
No, moved to run_solvent